code/game/mecha/mech_fabricator.dm

```diff
@@ -269,7 +269,7 @@
 	updateUsrDialog()
 	sleep(30) //only sleep if called by user
 
-	for(var/obj/machinery/computer/rdconsole/RDC in area_contents(get_area(src)))
+	for(var/obj/machinery/computer/rdconsole/RDC in oview(5,src))

```

Comment:
There are no setups where that happens. This is how the R&D setup works, so I am using oview here too.